lib/binary/binary.nit

```diff
@@ -187,6 +195,8 @@ redef abstract class Reader
 	#
 	# Using this format may result in a loss of precision as it uses less bits
 	# than Nit `Float`.
+	#
+	# Returns `0.0` when an error is pending (`last_error != null`).
```

Comment:
The "default" values were selected as to produce less side effects. `0.0` is neutral, whereas NaN is an exception in most algorithms. Also, serialization-side our JSON engines do not even support NaN, so I prefer to avoid it in this context.